sys/Makefile.dep

```diff
@@ -7,6 +7,14 @@ ifneq (,$(filter arduino,$(USEMODULE)))
   USEMODULE += $(SKETCH_MODULE)
 endif
 
+ifneq (,$(filter auto_init_actuators_sensors_%,$(USEMODULE)))
```

Comment:
What other than `auto_init_actuators_sensors_default` will this match?

Reply:
E.g. the auto init hook of the `sht1x` driver will be named `auto_init_actuators_sensors_sht1x`. And all other sensors and actuators should add their own module for auto-init as well, as otherwise individual selection of sensors and actuators to auto-init wouldn't work.

Reply:
Actually, it should ignore `auto_init_actuators_sensors_default`, so that if no sensor or actuator is auto-initalized, the function `auto_init_actuators_sensors()` is not added.